### DISCUSSION TITLE
Welcome to go-chatgpt-api Discussions!

### DISCUSSION THREAD

**Discussion by linweiyuan** [linweiyuan/go-chatgpt-api#43] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by dwdcth** [comment]
先来水一贴，非常感谢大佬的项目

**Reply by candychn** [reply]
感谢大佬，能不能给个简单的使用案例。

**Reply by linweiyuan** [reply]
案例在项目 example 文件夹

不会是零基础吧
